### PR TITLE
TATB- 32- Add footer section in all page 

### PR DESCRIPTION
**Pull request by PrabhatJTDeveloper**

### CODE REVIEW

File: src/app/component/header/header.component.html
Comment (PoojanPatelJT) [inline src/app/component/header/header.component.html:1]: Remove inline css for image tag and use class for it from image tag

File: src/app/component/header/header.component.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <nav class="navbar fixed-top navbar-expand-sm navbar-light bg-light" style="padding: 0px;">
   <div class="container-fluid">
     <a href="#" class="navbar-brand h1" routerLink="/dashboard" style="margin: 0px;">
-      <img class="d-inline-block" src="assets/img/logo1.png" width="35" height="35" />
+      <img class="d-inline-block" src="assets/img/logo1.png" alt="Jyoti Techn" width="35" height="35" />
```
Comment (PoojanPatelJT) [inline src/app/component/header/header.component.html]: Do not use h1 as className please remove it and give proper classname to it

**Review by PoojanPatelJT — CHANGES_REQUESTED** [review]

File: src/app/component/header/header.component.html (hunk 1 above)
Comment (charmijtdev) [reply src/app/component/header/header.component.html]: please move this inline css to external file

File: src/app/component/header/header.component.html [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <nav class="navbar fixed-top navbar-expand-sm navbar-light bg-light" style="padding: 0px;">
   <div class="container-fluid">
     <a href="#" class="navbar-brand h1" routerLink="/dashboard" style="margin: 0px;">
```
Comment (charmijtdev) [inline src/app/component/header/header.component.html]: why we have to add herf="#"

Comment (charmijtdev) [reply src/app/component/header/header.component.html]: also please change this inline css

**Review by charmijtdev — CHANGES_REQUESTED** [review]

File: src/app/component/header/header.component.html
Comment (PrabhatJTDeveloper) [reply src/app/component/header/header.component.html:1]: yes I have  Remove inline CSS an uses class for it

File: src/app/component/header/header.component.html (hunk 1 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/header/header.component.html]: yes I have  change inline CSS to external file

File: src/app/component/header/header.component.html (hunk 2 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/header/header.component.html]: ok I have Remove herf= "#"

**Comment by charmijtdev** [comment]
OLDPR

**Pull request closed without merging**